dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -127,6 +127,15 @@ def toggle_hidden
     render json: []
   end
 
+  def instructions
+    require_levelbuilder_mode
+    authorize! :read, ScriptLevel
+
+    script = Script.get_from_cache(params[:script_id])
+
+    render 'levels/instructions', locals: { stages: script.stages }
```

Comment:
Nit: elsewhere I think we prefer `@instance` vars over `local_assigns` for sharing data with the view.